packages/website/md/docs/asset_buyer/usage.md

```diff
@@ -0,0 +1,39 @@
+**Construction**
```

Comment:
Per convo at standup, I agree that we should speak a bit to the concept of best case and worst case somewhere in here.  

Should we also mention that this uses the forwarding contract, with a link and/or brief pitch about the advantages (no wrapping ETH or setting allowances)

Reply:
The `introduction.md` goes into the forwarder a bit and links out to the spec as well. I agree we can expand the topics covered but I would rather append an "advanced" section at the end of usage. I'll follow up after this PR